index.js

```diff
@@ -1,17 +1,71 @@
 export const getUserChoice = (userInput) => {
   // TODO
+  let res = userInput.trim().toLowerCase();
+  if (res === 'rock' || res === 'scissors' || res === 'paper' || res === 'bomb'){
+    return res;
+  } else {
+    return 'error';
+  }
 };
 
 export const getComputerChoice = () => {
   // TODO
+  // rock = 0;
+  // paper = 1;
+  // scissors = 2;
+  
+  let compChoice = Math.floor(Math.random() * 3);
+  if(compChoice === 0){
+    compChoice = 'rock';
+  } else if(compChoice === 1) {
+    compChoice = 'paper';
+  } else {
+    compChoice = 'scissors';
+  }
+  
+  return compChoice;
 };
 
 export const determineWinner = (userChoice, computerChoice) => {
   // TODO
+  if(userChoice === computerChoice) {
+    return 'Tie';
+  } else if(userChoice === 'rock'){
+      if(computerChoice === 'scissors'){
+        return 'You won';
```

Comment:
You should return exact string `user won` or `computer won`. Tests fail otherwise.